### PR TITLE
Drag and drop reordering for pipeline modules

### PR DESCRIPTION
**Pull request by jeffsyang**
@bethac07 @0x00b1

### CODE REVIEW

File: cellprofiler/gui/pipelinelistview.py [hunk 1]
```diff
@@ -1041,7 +1041,7 @@ def OnData(self, x, y, action):
                 self.data_object.GetReceivedFormat().GetType()
                 == self.pipeline_data_object.GetFormat().GetType()
             ):
-                pipeline_data = self.pipeline_data_object.GetDataHere()
+                pipeline_data = self.pipeline_data_object.GetData().tobytes().replace(b'\00', '')
```
Comment (AetherUnbound) [inline cellprofiler/gui/pipelinelistview.py]: I notice you're replacing a byte sequence `b'\00'` with a string - is `pipeline_data` supposed to be a string here or a `bytes`?

Comment (jeffsyang) [reply cellprofiler/gui/pipelinelistview.py]: i believe it should be a string. if that's the case, I'll just wrap it in a `str()`?

Comment (AetherUnbound) [reply cellprofiler/gui/pipelinelistview.py]: I think it's worth it for clarity!

**Comment by 0x00b1** [comment]
Thanks @jeffsyang. Please rebase in the future! I manually added the line in #5fe8dc768.

**Pull request closed without merging**